src/encoded/static/components/browse/components/file-tables.js

```diff
@@ -623,7 +653,7 @@ export class RawFilesStackedTable extends React.PureComponent {
      * Much of styling/layouting is defined in CSS.
      */
     render(){
-        const { experimentSet, columnHeaders: propColHeaders, showMetricsColumns, collapseLongLists, width, preventExpand } = this.props;
+        const { experimentSet, columnHeaders: propColHeaders, showMetricsColumns, showNotesColumns, collapseLongLists, width, preventExpand } = this.props;
```

Comment:
Codacy found an issue: ['showMetricsColumns' is missing in props validation](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5860998)